### PR TITLE
feat(www): show github stars

### LINKED ISSUE #46: [www] add github star count next to our github logo
Example:

image

### PR DESCRIPTION
**Pull request by todevmilen**
Fetch and Display GitHub Star Count in Nav section.
___ 
This pull request introduces changes to the githubStarButton.astro component to fetch and display the star count from a GitHub repository

Fixes #46

### CODE REVIEW

**Review by sammoore — CHANGES_REQUESTED** [review]
The changesets should be removed because this doesn't change the CLI and won't result in a new package to NPM.

**Comment by todevmilen** [comment]
> The changesets should be removed because this doesn't change the CLI and won't result in a new package to NPM.

Sorry for that. I removed the changeset.

File: www/src/components/landing/navbar/navbar.astro [hunk 1]
```diff
@@ -7,11 +7,21 @@ import GithubStarButton from "./githubStarButton.astro";
   class="z-[1] flex p-[0.5%] xl:p-[1%] 2xl:p-[1.5%] text-white justify-between items-center w-full"
 >
   <div class="m-2 h-10 w-10"></div>
+
```
Comment (danstepanov) [inline www/src/components/landing/navbar/navbar.astro:10]: @todevmilen can you remove this extra space?

Comment (todevmilen) [reply www/src/components/landing/navbar/navbar.astro]: Done :)

**Pull request merged**